src/app/src/main/kotlin/org/imperial/mrc/hint/models/VersionFileWithPath.kt

```diff
@@ -1,10 +1,10 @@
 package org.imperial.mrc.hint.models
 
-data class VersionFile(val hash: String, val filename: String, val fromADR: Boolean)
+data class VersionFile(val hash: String, val filename: String, val fromAdr: Boolean)
```

Comment:
Jooq no longer serialises from_adr to fromADR, probably because it's not a standard camel case. Making it a camel case works.